release_notes/v2.2.1.md

```diff
@@ -0,0 +1,160 @@
+v2.2.1 Release Notes - TBD, 2020
+================================
+
+What's New in Hyperledger Fabric v2.2.1
+---------------------------------------
+
+Fixes
+-----
+
+**FAB-18163: orderer certificate expiration - TLSHandshakeTimeShift without separate cluster port**
+
+If the TLS certificates of the ordering service nodes expire and are not replaced in time,
+communication between them cannot be established, and it will be impossible to send
+new transactions to the ordering service. To recover from such a scenario, it is possible
+to configure a backwards timeshift that ordering service nodes will utilize for TLS
+handshakes so that transactions can be processed.
+If the cluster service is sharing the orderer’s main gRPC server port,
+configure the new orderer `General.TLS.TLSHandshakeTimeShift` property.
+If using a separate cluster listener port,
+configure the orderer `General.Cluster.TLSHandshakeTimeShift` property.
+
+**FAB-18205: peer CLI certificate expiration - Permit peer CLI to communicate with orderers with expired TLS certs**
+
+The change allows peer CLI to communicate with ordering service nodes with expired TLS certificates
+by setting the `--tlsHandshakeTimeShift` flag to a desired backwards timeshift.
+The change applies to the `peer channel fetch` and `peer channel update` commands to allow
+fetching configuration blocks and updating the channel configuration for orderers with expired TLS certificates.
+
+**FAB-18171: orderer certificate expiration - Disregard certificate validity period in intra-orderer communication**
+
+This change makes the orderer cluster authentication infrastructure
+disregard validity periods when comparing certificates, and only regard public keys.
+With this change, an expiring Raft TLS certificate can be replaced
+with a new certificate that has the same public key, without issuing channel configuration updates.
+
+**FAB-18188: peer and orderer certificate expiration - Log expiration date upon startup**
+
+The future enrollment, tls server, and tls client certificate expiration dates are now logged upon peer and orderer startup.
```

Comment:
What does "the future" mean? 
Maybe: "The enrollment, tls server and client certificate expiration dates are now logged..." 
why do we need "the future" here? It's not necessarily in the future, it can be also in the past

Reply:
Done